web/src/main/java/org/cbioportal/web/config/WebServletContextListener.java

```diff
@@ -41,12 +44,11 @@ public void afterPropertiesSet() throws Exception {
     }
 
     private void checkOncokbInfo() {
```

Comment:
@jjgao I still remained the message. But let me know if you think it's no longer needed.